servicetest/networkgateway/test_network_gateway.py

```diff
@@ -110,13 +110,18 @@ def test_network_policy_drop(self):
         """
         try:
             sc = Container()
-            sc.start(DATA)
-            sc.set_gateway_config("network", GW_CONFIG_POLICY_DROP)
-            sc.launch_command("python " +
+            success = sc.start(DATA)
+            assert success is True
+            
+            success = sc.set_gateway_config("network", GW_CONFIG_POLICY_DROP)
+            assert success is True
+
+            pid, success = sc.launch_command("python " +
                               sc.get_bind_dir() +
                               "/testhelper.py" +
                               " --test-dir " + sc.get_bind_dir() +
                               " --do-ping example.com")
+            assert success is True
```

Comment:
I don't like all these asserts on the success value. That's not what the tests are supposed to test in most cases (except for the current caps test perhaps). Is the purpose to make it more clear that something has gone wrong in the test setup? Could that be made clear in some other way that doesn't clutter the tests?